ee/server/service/embedded_scripts/windows_lock.ps1

```diff
@@ -33,3 +33,18 @@ Get-LocalUser | Where-Object { $_.Enabled -eq $true } | ForEach-Object {
 }
 
 Write-Host "All users have been logged out and their accounts disabled."
+
+
+# Disable additional AD logins
+New-ItemProperty -Path "HKLM:\Software\Microsoft\PolicyManager\default\Settings\AllowSignInOptions" -Name 'value' -Value 3 -PropertyType DWORD -Force
+
+# Disable cached logins for AD/Azure/Entra accounts
+New-ItemProperty -Path "HKLM:\Software\Microsoft\Windows NT\CurrentVersion\Winlogon\" -Name 'CachedLogonsCount' -Value 0 -PropertyType String -Force
+
+Write-Host "All local non-administrative users have been logged out and their accounts disabled."
+Write-Host "Logging in with other Microsoft accounts has been disabled"
+Write-Host "Cached Logins have been disabled, disable the MDM-Enroled account to prevent further logins"
+
+# Shutdown computer in 10 seconds, after command has returned to fleet
```

Comment:
Nit: 10 -> 15 seconds

Reply:
Awesome thanks!

Reply:
this is also on the change over here: https://github.com/fleetdm/fleet/pull/20224/files

Reply:
Yep, fixed it there too